standards/code-style.adoc

```diff
@@ -344,18 +344,18 @@ if (condition) {
 
 [source, kotlin]
 ----
-if (condition)
+if (condition1 && condition2)
     body()  // WRONG!
 ----
 
-Braces are not required for `when` branches and `if` statement bodies if they meet BOTH
-the following conditions:
+Occasionally, statements are simple enough that Braces may be omitted. This should only
+be done if the logic is unmistakable and doing so increases readability of the method.
+You may _consider_ omitting braces if the situation meets the following conditions:
```

Comment:
I feel like we somehow got more proscriptive with this wording, rather than going a more guidance based approach like we talked about. 

Maybe that's just my reading, though?